pkg/apis/eventing/v1alpha1/trigger_validation.go

```diff
@@ -38,7 +38,7 @@ func (ts *TriggerSpec) Validate() *apis.FieldError {
 	}
 
 	if ts.Filter.SourceAndType == nil {
-		fe := apis.ErrMissingField("filter.exactMatch")
+		fe := apis.ErrMissingField("filter.sourceAndType")
```

Comment:
included it as part of this PR, although it has nothing to do :)
very cosmetic change